code/HUXt.py

```diff
@@ -346,6 +352,16 @@ def solve(self, cme_list=[], save=False, tag=''):
         :param tag: Identifying string to append to the filename.
         """
         
+        constants= huxt_constants()
+        alpha = constants['alpha']  # Scale parameter for residual SW acceleration
+        r_accel = constants['r_accel']  # Spatial scale parameter for residual SW acceleration
+        buffertime=constants['buffertime']
```

Comment:
Similar to elsewhere, i think we can just refer to the HUXt1D attributes of these, and we should add buffertime into the attributes at the __init__